plugins/katello/nightly/user_guide/activation_keys/index.md

```diff
@@ -86,24 +90,19 @@ Finally, when 'Auto-Attach' is disabled, all subscriptions on the activation key
 
 ![Adding subscriptions to an Activation Key](/plugins/katello/{{ page.version }}/user_guide/activation_keys/activation_key_add_subscriptions.png)
 
-
-
 ## Change Repository Enablement for an Activation Key
 
 To change repository enablement settings using a key:
 
 - navigate to: Content > Activation Keys
 - select the desired key from the list
-- click **Product Content**
-- click the edit icon for the repository content set that you would like to modify
-- select the desired value (e.g. 'Override to Yes', 'Override to No', 'Defaults to Yes', 'Defaults to No')
-- click **Save**
+- click **Repository Sets**
+- select the repository that you would like to modify
```

Comment:
s/select the repository that you would like to modify/select the repository you want to modify/  
{less words, easier to translate}